tests/beta/test_default_qubit_tf.py

```diff
@@ -869,7 +843,7 @@ class TestPassthruIntegration:
     """Tests for integration with the PassthruQNode"""
 
     def test_jacobian_variable_multiply(self, tol):
-        """Test that qnode.jacobian applied to the tensornet.tf device

```

Comment:
:-1: 
(sorry, just wanted to use that emoji one time)

Reply:
Feeling more and more that we should put "unit" tests next to the code, and "integration" tests in the test folder (and allow all plugins to be checked against same integration tests)